Apps/WebClient/src/ClientApp/src/views/timeline.vue

```diff
@@ -174,42 +174,68 @@
                         <NoteTimelineComponent :is-add-mode="true" />
                     </b-col>
                 </b-row>
-                <b-row class="view-selector justify-content-end">
-                    <b-col cols="auto" class="pr-0">
-                        <b-btn
-                            class="month-view-btn btn-outline-primary px-2 m-0"
-                            :class="{ active: !isListView }"
-                            @click.stop="toggleMonthView"
-                        >
-                            Month
-                        </b-btn>
-                    </b-col>
-                    <b-col cols="auto" class="pl-0">
-                        <b-btn
-                            class="list-view-btn btn-outline-primary px-2 m-0"
-                            :class="{ active: isListView }"
-                            @click.stop="toggleListView"
-                        >
-                            List
-                        </b-btn>
-                    </b-col>
-                </b-row>
                 <LinearTimeline
                     v-show="isListView && !isLoading"
                     :timeline-entries="timelineEntries"
                     :is-visible="isListView"
                     :total-entries="getTotalCount()"
                     :filter-text="filterText"
                     :filter-types="filterTypes"
-                />
+                >
+                    <b-row
+                        slot="month-list-toggle"
+                        class="view-selector justify-content-end"
+                    >
+                        <b-col cols="auto" class="pr-0">
+                            <b-btn
+                                class="month-view-btn btn-outline-primary px-2 m-0"
+                                :class="{ active: false }"
+                                @click.stop="toggleMonthView"
+                            >
+                                Month
+                            </b-btn>
+                        </b-col>
+                        <b-col cols="auto" class="pl-0">
+                            <b-btn
+                                class="list-view-btn btn-outline-primary px-2 m-0"
+                                :class="{ active: true }"
+                            >
+                                List
+                            </b-btn>
+                        </b-col>
+                    </b-row>
+                </LinearTimeline>
                 <CalendarTimeline
                     v-show="!isListView && !isLoading"
                     :timeline-entries="timelineEntries"
                     :is-visible="!isListView"
                     :total-entries="getTotalCount()"
                     :filter-text="filterText"
                     :filter-types="filterTypes"
-                />
+                >
+                    <b-row
```

Comment:
per my screenshot, this b-row can be well integrated and displayed on the same line as the MonthYear picker. This b-row allows us to have the same CSS for this Month List toggle in the same TimeLine.vue file.

Reply:
+1, looks like is being wrapped internally by a col